analyzer.js

```diff
@@ -27,6 +27,7 @@ const IGNORE_DIRS = process.env.ASTGEN_IGNORE_DIRS
       "flow-typed",
       "i18n",
       "__tests__",
+      "__mocks__",
```

Comment:
Do we need both now that we are checking for double underscore below? Can we remove line 29 as well?